tests/test_fips.py

```diff
@@ -130,8 +146,6 @@ def openssl_fips_hashes_test_fnct(container_per_test: ContainerData) -> None:
         ), f"unexpected digest of hash {digest}: {dev_null_digest}"
 
 
-@pytest.mark.parametrize(
-    "container_per_test", CONTAINERS_WITH_ZYPPER, indirect=True
-)
+@pytest.mark.parametrize("container_per_test", CONTAINER_IMAGES, indirect=True)
```

Comment:
This will fail, as the containers without zypper are not guaranteed to have openssl